### DISCUSSION TITLE
Is it possible to use a already installed bare-metal machine in a cluster?

### DISCUSSION THREAD

**Discussion by silenium-dev** [syself/cluster-api-provider-hetzner#702] [Q&A]
I'd like to use CoreOS for my nodes, which isn't supported to be installed with `installimage`.
So I went ahead and wrote an ansible playbook to deploy and set up my bare-metal machine with CoreOS and all required settings/tools.
I didn't find any documentation on using these set up machines in a cluster whatsoever, and the HetznerBareMetalMachine requires a config for `installimage`.

Is there any way to use CoreOS (either already set up or via `installimage`) with the CAPH?

**Comment by batistein** [comment]
You can specify any custom image in installimage, you just need to create a tarball of it and save it to an FTP server, for example. See more about how to create an image and what types of storage are supported: https://docs.hetzner.com/robot/dedicated-server/operating-systems/installing-custom-images/

**Reply by silenium-dev** [reply]
In the documentation it says, the custom image "must be one of Debian, Ubuntu, OpenSUSE or CentOS to get the network settings properly configured". CoreOS a special variant of Fedora with an ostree based root filesystem, and uses systemd-networkd instead of NetworkManager. I will try it though.

**Reply by batistein** [reply]
Sorry, as i have never used coreos with installimage i don't know. But I would ask here: https://github.com/hetzneronline/installimage or directly the Hetzner support

**Reply by silenium-dev** [reply]
I've opened an issue here: https://github.com/hetzneronline/installimage/issues/72
